presentation/src/main/java/com/kk/presentation/UserQuestionButtonView.kt

```diff
@@ -0,0 +1,58 @@
+package com.kk.presentation
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.kk.designsystem.R.drawable.main_button
+import com.kk.designsystem.components.KkBody
+import com.kk.designsystem.components.KkOrangeTitle
+import com.kk.designsystem.components.KkTitle
+
+@Composable
+fun UserQuestionButtonView() {
+    val image = painterResource(id = main_button)
+    Column(
+        modifier = Modifier
+            .verticalScroll(rememberScrollState())
+            .fillMaxSize()
+            .padding(horizontal = 30.dp)
+    ) {
+        KkTitle(
+            label = stringResource(R.string.uqb_title),
+            modifier = Modifier
+                .align(Alignment.CenterHorizontally)
+                .padding(vertical = 40.dp)
+        )
+        Spacer(modifier = Modifier.size(50.dp))
+        KkOrangeTitle(
+            label = stringResource(R.string.uqb_time),
+            modifier = Modifier.align(Alignment.CenterHorizontally)
+        )
+        Spacer(modifier = Modifier.size(40.dp))
+        Image(
+            painter = image,
+            contentDescription = "button_main",
```

Comment:
This description you have to create a variable in a String file

Reply:
Fixed